code/modules/mob/emote.dm

```diff
@@ -82,10 +81,19 @@
 	mob_type_allowed_typecache = list(/mob/living, /mob/dead/observer, /mob/camera/imaginary_friend)
 	mob_type_ignore_stat_typecache = list(/mob/dead/observer, /mob/living/silicon/ai, /mob/camera/imaginary_friend)
 
+// BUBBER EDIT CHANGE BEGIN - Flip Cooldown
 /datum/emote/flip/run_emote(mob/user, params , type_override, intentional)
 	. = ..()
+	var/mob/living/carbon/flippy_mcgee = user
+	if(flippy_mcgee)
```

Comment:
Casting a different type won't make it null, it'll just runtime later. A byondism I've learned the hard way